indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -88,8 +87,8 @@ def _process_DELETE(self):
         if EditingRevisionFile.query.with_parent(self.file_type).has_rows():
             raise UserValueError(_('Cannot delete file type which already has files'))
 
-        review_conditions = editing_settings.get(self.event, self.editable_type.name + '_review_conditions')
-        if any(self.file_type.id in cond for cond in review_conditions.values()):
+        review_conditions = EditingReviewCondition.query.with_parent(self.event).filter_by(type=self.editable_type)
+        if any(self.file_type in review_conditions.file_types):
```

Comment:
you can simplify this a lot now by checking `if self.file_type.review_conditions`